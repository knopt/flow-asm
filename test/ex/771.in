32
1
-6.5466378235849865
4.8664067147442065 1.612792209679899 4.467636723874506 -2.9872119366209766 -1.686727794954102 1.595664263284398 -8.621363262921392 6.447623049603187 -9.616081970682957 5.574466924435768 -9.908160496771623 5.8846106784420265 6.941574498057747 -7.89749957695947 -0.88023704363167 -4.274177845482214 -5.448610948990038 6.766467037303617 -4.113862819270572 -5.575341528174313 -0.8788607346757544 6.239322370415913 5.088204382368172 -0.03067285723038715 3.332168820091221 -6.0002194633921935 5.563316364250365 -3.916491141898626 7.829761270433401 -1.2311848447560632 -2.399998097937668 -9.310491269953276 

9.785618091312593 
