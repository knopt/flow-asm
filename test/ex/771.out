-27.337888545307955 -19.687443572538843 23.157269800955024 -51.791406146522576 6.827070875854716 23.084296263583536 -75.50854211058916 105.09881880626105 -114.77934084340025 105.02148888223714 -111.26731478092202 109.27416379331842 13.861133817714883 -105.04354318344659 45.059239275077914 -26.493079069885344 -13.13719912568466 86.73415860041398 -75.34344179140018 -15.143113322247693 29.86729806578234 52.83948912141265 -2.4477481775942174 -33.54210820827462 25.34747533957284 -67.09598558442609 81.26559738787526 -65.97735752195847 84.72822159838265 -60.54991739992086 -10.051795149923777 -54.55098724949645 
